packages/manager/modules/anthos/src/dashboard/host/order/anthos-dashboard-host-order.html

```diff
@@ -0,0 +1,129 @@
+<div>
+    <oui-back-button data-on-click="::$ctrl.goBack()"></oui-back-button>
+    <oui-header
+        class="mb-0"
+        data-heading="{{:: 'anthos_dashboard_host_order_header_heading' | translate}}"
+        data-description="{{:: 'anthos_dashboard_host_order_header_description' | translate}}"
+    ></oui-header>
```

Comment:
Maybe you can use this component variant: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-back-button-webcomponent--with-heading?

Reply:
Why not but what about the description ?

Reply:
You can have the description set directly as a child of the element

Reply:
Seen together. Fixed